tests/web/render.rs

```diff
@@ -35,6 +35,41 @@ fn container_is_emptied_upon_drop() {
     assert!(container.first_child().is_none());
 }
 
+/// Renders a child with a lifetime scoped to the RenderContext bump arena.
+#[wasm_bindgen_test]
+fn render_bump_scoped_node() {        
+    struct Child<'a> {
+        name: &'a str,
+    }
+
+    impl<'a> Render<'a> for Child<'a> {
+        fn render(&self, _cx: &mut RenderContext<'a>) -> Node<'a> {
+            text(self.name)
+        }
+    }
+
+    struct Parent;
+
+    impl<'a> Render<'a> for Parent {
+        fn render(&self, cx: &mut RenderContext<'a>) -> Node<'a> {
+            let child_name = String::from_str_in("child", cx.bump).into_bump_str();
+
+            div(&cx)
+                .children([Child { name: child_name }.render(cx)])
+                .finish()
+        }
+    }
+
+    let parent = Rc::new(RenderFn(|cx| {
+        Parent.render(cx)
+    }));
+
+    let container = create_element("div");
+    let _vdom = Vdom::new(&container, parent.clone());
+
+    assert_rendered(&container, &parent);
+}
+
```

Comment:
Here's the smoke test. It's pretty similar to the other one, but I used the `assert_rendered` afterward